### ISSUE TITLE
Update Guava library

### ISSUE THREAD

**Issue by tonihele** [Simsilica/SiO2#15]
Update Guava library. Lots of warnings from vulnerabilities.

https://mvnrepository.com/artifact/com.google.guava/guava/19.0

**Comment by pspeed42** [comment]
Yes, I need to update... but note that if you include a newer one in your dependencies then that is the one that will be used.

guava-19 is the "minimum compatible version" that SiO2 supports.  In your own projects, you are free to use the next highest version without breaking changes.

And if there is a next highest (and desired) version that has breaking changes that affect SiO2 then I definitely want to know as that will up the priority of this request.

**Comment by tonihele** [comment]
My projects don't use Guava. Only comes from SiO2 dependency. So I didn't play around with any versions to see whether they work or not.
